scripts/txt2img.py

```diff
@@ -55,14 +55,18 @@ def __create_inference_pipe(unet, text_encoder, tokenizer, scheduler, vae):
         tokenizer=tokenizer,
         unet=unet,
         scheduler=scheduler,
-        safety_checker=None, # save vram
-        requires_safety_checker=None, # avoid nag
-        feature_extractor=None, # must be none of no safety checker
+        safety_checker=None,  # save vram
+        requires_safety_checker=None,  # avoid nag
+        feature_extractor=None,  # must be none of no safety checker
     )
 
     return pipe
 
+
 def main(args):
+    # Create output folder if it doesn't exist
+    os.makedirs('output', exist_ok=True)
```

Comment:
Figured it'd be neater to not have the generated images next to the source code.